assets/blocks/reader-revenue-manager/common/ButtonEdit.js

```diff
@@ -76,7 +76,7 @@ export default function ButtonEdit( {
 				).hasModuleAccess( 'reader-revenue-manager' );
 			}
 
-			setHasModuleAccess( hasModuleOwnershipOrAccess );
+			setHasModuleAccess( !! hasModuleOwnershipOrAccess );
```

Comment:
Let's add a comment here to clarify this:

```suggestion
			// Note: `hasModuleOwnershipOrAccess` can be expected to be undefined if
			// - `ownerID` is not set for view-only or non-SK users.
			// - the `check-access` endpoint returns a 403 error for non-SK users.
```